Code/Framework/AzCore/Tests/Debug/PerformanceCollectorTests.cpp

```diff
@@ -0,0 +1,215 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzCore/UnitTest/UnitTest.h>
+#include <AzTest/AzTest.h>
+
+#include <AzCore/Debug/PerformanceCollector.h>
+#include <AzCore/JSON/document.h>
+
+using namespace AZ;
+using namespace Debug;
```

Comment:
using namespace at global scope is not allowed per the [Coding Standard](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#namespaces).

This must be removed and `using namespace`  should be avoided in general

Reply:
Yeah, copy/paste leftover. Removed.